### ISSUE TITLE
Create the splitting page

### ISSUE THREAD

**Issue by ticoann** [dmwm/WMCore#6594]
splitting page  (only editable before request is assigned.
Something like following.
https://cmsweb-testbed.cern.ch/reqmgr/view/splitting/amaltaro_TaskChain_MC_PhEDExSingleThread_v1_160209_165949_6091
Current template
https://github.com/dmwm/WMCore/blob/master/src/templates/WMCore/WebTools/RequestManager/Splitting.tmpl

But as we discussed json format will be fine

This request is not urgent

**Comment by ticoann** [comment]
@vkuznet, Valentin, we can discuss more detail tomorrow after the meeting. #6596 need to be done first

**Comment by ticoann** [comment]
For config format,
https://reqmgr2-dev.cern.ch/reqmgr2/data/workload_config?name=sryu_MonteCarlo_eff_wq_testt_160217_212435_2532

For splitting
https://reqmgr2-dev.cern.ch/reqmgr2/data/splitting?name=sryu_MonteCarlo_eff_wq_testt_160217_212435_2532

**Comment by ticoann** [comment]
Current reqmgr1 page which include splitting button
https://cmsweb-testbed.cern.ch/reqmgr/view/details/dmason_BoogaBoogaDcache_160215_060715_6989

When click job splitting button

https://cmsweb-testbed.cern.ch/reqmgr/view/splitting/dmason_BoogaBoogaDcache_160215_060715_6989


Corresponding reqmgr2 page which doesn't have job splitting button.
https://cmsweb-testbed.cern.ch/reqmgr2/fetch?rid=dmason_BoogaBoogaDcache_160215_060715_6989

1. add splitting button  when it gets clicked (API need to be used to get the data - GET call) 
   
https://reqmgr2-dev.cern.ch/reqmgr2/data/splitting/sryu_MonteCarlo_eff_wq_testt_160217_212435_2532  

display editable json format  like following
{
"splitAlgo": "EventBased",
"splitParams": 
{
"algorithm": "EventBased",
"events_per_lumi": ​250,
"events_per_job": ​500,
"deterministicPileup": false,
"lheInputFiles": false,
"trustSitelists": false,
"type": "Production"
},
"taskName": "/sryu_MonteCarlo_eff_wq_testt_160217_212435_2532/Production",
"taskType": "Production"
},
{
"splitAlgo": "MinFileBased",
"splitParams": 
{
"trustSitelists": false,
"type": "LogCollect",
"files_per_job": ​500,
"algorithm": "MinFileBased"
},
"taskName": "/sryu_MonteCarlo_eff_wq_testt_160217_212435_2532/Production/LogCollect",
"taskType": "LogCollect"
},

2. for submission  of update data use the same call with POST (pass json record as body)
https://reqmgr2-dev.cern.ch/reqmgr2/data/splitting/sryu_MonteCarlo_eff_wq_testt_160217_212435_2532

3. when submit is successful display updated parameter

4. It is only editable if request is new, assigned-approved status.

**Comment by vkuznet** [comment]
Seangchan,
we discussed the splitting page and I already have code for it. But we didn't
explicitly discuss the config format use case
> For config format,
> https://reqmgr2-dev.cern.ch/reqmgr2/data/workload_config?name=sryu_MonteCarlo_eff_wq_testt_160217_212435_2532

Please elaborate more where this call should be done. I need a pointer on web
page where I'll plug a call to display config.
Best,
Valentin.

**Comment by ticoann** [comment]
actually I created separate ticket for config #6599 :smile:

**Comment by vkuznet** [comment]
Seangchan, please review this PR: https://github.com/dmwm/WMCore/pull/6605 and the only missing part would be implementation of PUT ReqMgr2 API for splitting parameters
